lib/ButtonLink.js

```diff
@@ -1,6 +1,6 @@
-var React = require('react');
+var React = require('react/addons');
```

Comment:
This will apply all the React addons to this library which is more than it needs. Try `require('react/lib/cx');` instead to only pull in the classSet addition. This should apply to all these files.